data/explorations/all_interactions/all_interactions.yaml

```diff
@@ -629,6 +630,21 @@ states:
           rule_type: DoesNotContainAtLeastOneOf
         tagged_misconception_id: null
         training_data: []
+      - outcome:
+          dest: Item Selection Multi
+          feedback:
+            content_id: feedback_4
+            html: <p>That's good!, no bad options, but choose all good options!</p>
```

Comment:
Weird punctuation. Drop the first comma and change "no" to "No".

Also consider changing "choose" to "make sure to choose" and "good options" to "the good options".

Reply:
changed it